packages/dd-trace/src/plugins/tracing.js

```diff
@@ -94,7 +94,7 @@ class TracingPlugin extends Plugin {
   }
 
   addError (error, span = this.activeSpan) {
-    if (span && !span._spanContext._tags.error) {
+    if (!span.context()._tags.error) {
```

Comment:
Why remove this check that the span is truthy?

Reply:
Leftover from a previous design iteration, i reverted this code change.